symengine/lib/symengine_wrapper.pyx

```diff
@@ -689,7 +689,7 @@ cdef class Basic(object):
         elif (op == 5):
             return c2py(<RCP[const symengine.Basic]>(symengine.Ge(A.thisptr, B.thisptr)))
 
-    def expand(Basic self not None):
+    def expand(Basic self not None, *args, **kwargs):
```

Comment:
The actual issue that I was facing was that during the use of `exp` in `SymPy`, the associated object was making like:
```
e.expand(complex=True)
```
I don't think the expansion for complex domain has been implemented in `SymEngine` for now. What do you suggest we do?